src/Ledger/Conway/Conformance/Certs.agda

```diff
@@ -123,7 +123,7 @@ data _⊢_⇀⦇_,DELEG⦈_ where
           )
     ∙ mkh ∈ mapˢ just (dom pools) ∪ ❴ nothing ❵
       ────────────────────────────────
-      ⟦ pp , pools , delegatees ⟧ᵈᵉ ⊢
+      ⟦ pp , pools , delegatees ⟧ ⊢
       ⟦ vDelegs , sDelegs , rwds , dep ⟧ᵈ
```

Comment:
Before this point in the file, you could add the following:
```
instance
  unquoteDecl To-CertEnv To-DState To-PState To-GState To-CertState To-DelegEnv = derive-To
    (   (quote CertEnv , To-CertEnv)
    ∷   (quote DState , To-DState)
    ∷   (quote PState , To-PState)
    ∷   (quote GState , To-GState)
    ∷   (quote CertState , To-CertState)
    ∷ [ (quote DelegEnv , To-DelegEnv) ])
```
and then you could remove the superscript on line 127 as well as similar suggestions below.

```suggestion
      ⟦ vDelegs , sDelegs , rwds , dep ⟧
```